README.md

```diff
@@ -887,14 +908,17 @@ The wallet proxy must have access to
   and anonymity revokers, and their public keys and metadata.
 - optionally, private keys of the CCD drop account. If this is not provided then
   the gtu drop functionality will be disabled.
+- optionally, the configuration file for the `v0/appSettings` endpoint. See
+  [Forced update configuration file](#forced-update-configuration-file) for the format of the file.
 
 An example invocation is
 ```console
 wallet-proxy --grpc-ip 127.0.0.1\ # IP of the node
```

Comment:
This style of commenting doesn't work, at least not in bash; the backshash has to escape the newline. The correct way to do it is quite hideous: https://stackoverflow.com/a/12797512.